CMSIS/Core/Include/cmsis_compiler.h

```diff
@@ -278,6 +278,20 @@
   #error Unknown compiler.
 #endif
 
+/*
+ * @brief Function to extract two 8-bit values from an operand (op1) rotated by 8/16/24 bits (rotate) and sign-extend them to 16 bits each
+ * @param[in] op1     input operand
+ * @param[in] rotate  number of bits (allowed 8/16/24) to rotate the input operand
+ * @return            the 8-bit values sign-extended to 16-bit values
+ */
+__STATIC_FORCEINLINE uint32_t __SXTB16_RORn(uint32_t op1, uint32_t rotate)
```

Comment:
We don't have any function definitions in this selector-header, yet. Can we move this function next to the relating ones into the specific compiler headers, please?

Reply:
Hi @JonatanAntoni ,

sorry for my late reply but I was traveling last week and I did not check the comments on github. Does this mean that I need to move this function to cmsis_gcc.h ?

Reply:
Yes, and probably add it to the other compiler headers as well.

Reply:
Hi @JonatanAntoni, I have added the support in the compilers as suggested